js/src/compatability/compatability.js

```diff
@@ -0,0 +1,68 @@
+/* External dependencies */
```

Comment:
Perhaps call the file CompatibilityHelper as well, as that is the name of the class (or vice versa).